db/seeds/searchSeed.js

```diff
@@ -0,0 +1,51 @@
+var Search=require('../models/model');
+const mongoose =require('mongoose');
+mongoose.connect('mongodb://localhost/fetcher');
+const db = mongoose.connection;
+
+mongoose.connection.collections['searches'].drop( function(err) {

```

Comment:
`Search.deleteMany({});` would be more concise and would still remove all entries from the searches collection. Note: you'd still need a callback or some other way to handle the asynchronous nature of `deleteMany` if async operations there would affect the rest of your code.

Reply:
duly noted